launcher/translations/TranslationsModel.cpp

```diff
@@ -527,34 +528,34 @@ Language * TranslationsModel::findLanguage(const QString& key)
     }
 }
 
+void TranslationsModel::setUseSystemLocale(bool useSystemLocale)
+{
+    APPLICATION->settings()->set("UseSystemLocales", useSystemLocale);
```

Comment:
I think singular makes more sense, though discard this if you think otherwise.
```suggestion
    APPLICATION->settings()->set("UseSystemLocale", useSystemLocale);
```